lib/stellar_core_commander/docker_process.rb

```diff
@@ -13,8 +23,8 @@ def required_ports
 
     Contract None => Any
     def launch_state_container
-      $stderr.puts "launching state container #{state_container_name}"
-      docker %W(run --name #{state_container_name} -p #{postgres_port}:5432 --env-file stellar-core.env -d stellar/stellar-core-state)
+      $stderr.puts "launching state container #{state_container_name} from image #{@docker_state_image}"
```

Comment:
Seems like this "from" info would be good to put in launch_stellar_core as well.